src/parser.rs

```diff
@@ -391,7 +394,16 @@ impl<'tokens, 'src> Parser<'tokens, 'src> {
 
   /// Parse an expression, e.g. `1 + 2`
   fn parse_expression(&mut self) -> CompileResult<'src, Expression<'src>> {
-    if self.accepted_keyword(Keyword::If)? {
+    if self.depth == 64 {
```

Comment:
How about we just drop it down for windows:

```suggestion
    if self.depth == if cfg!(windows) { 64 } else { 256 } {
```

Reply:
this change is in the newest push (though 255 not 256)